site/app/views/grading/ElectronicGraderView.php

```diff
@@ -105,13 +111,19 @@ public function statusPage(
             else {
                 $total_students = $total_submissions;
             }
-            $num_components = count($gradeable->getNonPeerComponents());
+            $num_peer_components = 0;
+            $ta_graded_percentage = 0;
```

Comment:
This isn't needed, you set $ta_graded_percentage to 0 on line 74 and it hasn't been modified between then and here

Reply:
Updated percentage calculation in recent commits and removed using $ta_graded_percentage at all.